nncf/parameters.py

```diff
@@ -89,6 +89,19 @@ class CompressWeightsMode(Enum):
     INT8 = "int8"  # Deprecated mode
 
 
+class MixedPrecisionMode(Enum):
```

Comment:
`MixedPrecisionMode` sounds too generic. How about `PrecisionSelectionMode`, `SensitivityMode`, `SensitivityMetric`?

Reply:
Renamed to SensitivityMetric